GitTrends/Views/Repository/InformationButton.cs

```diff
@@ -33,16 +33,19 @@ public InformationButton(MobileSortingService mobileSortingService, in IMainThre
 
-            Children.Add(new FloatingActionTextButton(mobileSortingService, FloatingActionButtonSize.Mini, FloatingActionButtonType.Statistic1).Center().Assign(out _statistic1FloatingActionButton)
+            Children.Add(new FloatingActionTextButton(mobileSortingService, FloatingActionButtonType.Statistic1, floatingActionButtonSize: FloatingActionButtonSize.Mini).Center().Assign(out _statistic1FloatingActionButton)
                             .Bind<FloatingActionTextButton, IReadOnlyList<Repository>, string>(FloatingActionTextButton.TextProperty, nameof(RepositoryViewModel.VisibleRepositoryList), BindingMode.OneWay, convert: repositories => StatisticsService.GetFloatingActionTextButtonText(mobileSortingService, repositories, FloatingActionButtonType.Statistic1)));
 
-            Children.Add(new FloatingActionTextButton(mobileSortingService, FloatingActionButtonSize.Mini, FloatingActionButtonType.Statistic2).Center().Assign(out _statistic2FloatingActionButton)
+            Children.Add(new FloatingActionTextButton(mobileSortingService, FloatingActionButtonType.Statistic2, floatingActionButtonSize: FloatingActionButtonSize.Mini).Center().Assign(out _statistic2FloatingActionButton)
                             .Bind<FloatingActionTextButton, IReadOnlyList<Repository>, string>(FloatingActionTextButton.TextProperty, nameof(RepositoryViewModel.VisibleRepositoryList), BindingMode.OneWay, convert: repositories => StatisticsService.GetFloatingActionTextButtonText(mobileSortingService, repositories, FloatingActionButtonType.Statistic2)));
 
-            Children.Add(new FloatingActionTextButton(mobileSortingService, FloatingActionButtonSize.Mini, FloatingActionButtonType.Statistic3).Center().Assign(out _statistic3FloatingActionButton)
+            Children.Add(new FloatingActionTextButton(mobileSortingService, FloatingActionButtonType.Statistic3, floatingActionButtonSize: FloatingActionButtonSize.Mini).Center().Assign(out _statistic3FloatingActionButton)
                             .Bind<FloatingActionTextButton, IReadOnlyList<Repository>, string>(FloatingActionTextButton.TextProperty, nameof(RepositoryViewModel.VisibleRepositoryList), BindingMode.OneWay, convert: repositories => StatisticsService.GetFloatingActionTextButtonText(mobileSortingService, repositories, FloatingActionButtonType.Statistic3)));
 
-            Children.Add(new FloatingActionTextButton(mobileSortingService, FloatingActionButtonSize.Normal, FloatingActionButtonType.Information, new AsyncCommand(ExecuteFloatingActionButtonCommand)) { FontFamily = FontFamilyConstants.RobotoMedium, Text = "TOTAL" }.Center().Assign(out _totalButton));
+            Children.Add(new FloatingActionTextButton(mobileSortingService, FloatingActionButtonType.Information, new AsyncCommand(ExecuteFloatingActionButtonCommand)) { FontFamily = FontFamilyConstants.RobotoMedium/*, Text = "IOMLAN"*/ }.Center().Assign(out _totalButton)
```

Comment:
Let's remove the comment before we merge it into the main branch: `/*, Text = "IOMLAN"*/`